Makefile

```diff
@@ -64,9 +64,11 @@ flake8:
 
 .PHONY: unit
 unit: gitdeps
+	mkdir -p .coverage || true
```

Comment:
question: What's the motivation behind `|| true` here? I'm trying to imagine scenarios where `mkdir -p` would fail that we'd like to ignore.

Reply:
I saw a failure at one point while debugging and I can't reproduce it now so I'm removing the `|| true`, thanks for flagging it.

Reply:
Ahh that makes sense. Sounds good to me, thanks!